source/introduction-to-information-technology/it-info-prerequisite.md

```diff
@@ -49,7 +49,7 @@ Moreover, the timing shall be updated on the following timetable page:
 ### Course prerequisites : 
 
 
-- This course is for very beginners and only require that you have the basic Windows or Linux operating system as a normal user.
```

Comment:
Would you please keep this change (very beginners) as it is?
We mean it, this course is for very beginners.

Reply:
@ShereenS or if you have a better description that explain our meaning, please share.

Reply:
@ShereenS This good idea but would you please add that it required basic programing, networking, DB, PowerShell and Bash script as well.
Thanks!

Reply:
@MohamedRadwan-DevOps done